src/OSPSuite.R/Services/DataImporter.cs

```diff
@@ -0,0 +1,198 @@
+         organCategory.TopNames.Add(Constants.ObservedData.PERIPHERAL_VENOUS_BLOOD_ORGAN);
+         organCategory.TopNames.Add(Constants.ObservedData.VENOUS_BLOOD_ORGAN);
+         categories.Add(organCategory);
+
+         var compCategory = createMetaDataCategory<string>(Constants.ObservedData.COMPARTMENT, isMandatory: true, isListOfValuesFixed: true);
+         compCategory.Description = ObservedData.ObservedDataCompartmentDescription;
+         compCategory.TopNames.Add(Constants.ObservedData.PLASMA_COMPARTMENT);
+         categories.Add(compCategory);
+
+         var moleculeCategory = createMetaDataCategory<string>(Constants.ObservedData.MOLECULE);
+         moleculeCategory.Description = ObservedData.MoleculeNameDescription;
+         moleculeCategory.AllowsManualInput = true;
+         categories.Add(moleculeCategory);
+
+         // Add non-mandatory metadata categories
+         var molecularWeightCategory = createMetaDataCategory<double>(Constants.ObservedData.MOLECULARWEIGHT);
+         molecularWeightCategory.MinValue = 0;
+         molecularWeightCategory.MinValueAllowed = false;
+         categories.Add(molecularWeightCategory);
+         categories.Add(createMetaDataCategory<string>(Constants.ObservedData.STUDY_ID));
+         categories.Add(createMetaDataCategory<string>(Constants.ObservedData.SUBJECT_ID));
+         categories.Add(createMetaDataCategory<string>(Constants.ObservedData.GENDER, isListOfValuesFixed: true));
+         categories.Add(createMetaDataCategory<string>(Constants.ObservedData.DOSE));
+         categories.Add(createMetaDataCategory<string>(Constants.ObservedData.ROUTE));
+         categories.Add(createMetaDataCategory<string>(Constants.ObservedData.GROUP_ID));
+
+         return categories;
+      }
+
+      public (IReadOnlyList<DataRepository> DataRepositories, ImporterConfiguration Configuration) ImportDataSets(
+         IReadOnlyList<MetaDataCategory> metaDataCategories, 
+         IReadOnlyList<ColumnInfo> columnInfos, 
+         DataImporterSettings dataImporterSettings,
+         string dataFileName)
+      {
+         throw new NotImplementedException();
+      }
+
+      public IReadOnlyList<DataRepository> ImportFromConfiguration(
+         ImporterConfiguration configuration, 
+         IReadOnlyList<MetaDataCategory> metaDataCategories, 
+         IReadOnlyList<ColumnInfo> columnInfos, 
+         DataImporterSettings dataImporterSettings,
+         string dataFileName)
+      {
+         if (string.IsNullOrEmpty(dataFileName))
+            return Enumerable.Empty<DataRepository>().ToList();
+
+         try
+         {
+            var importedData = _importer.ImportFromConfiguration(configuration, columnInfos, dataFileName, metaDataCategories, dataImporterSettings);
+            if (importedData.MissingSheets.Count != 0)
+               _dialogCreator.MessageBoxError(Captions.Importer.SheetsNotFound(importedData.MissingSheets));
```

Comment:
why does the function show error message boxes instead of throwing exceptions?
I would expect that the function is called in silent mode in R, e.g. later as part of a qualification workflow (which later on will be based on **referenced** observed data)

Reply:
In this case, I would not like to throw an exception. This means that some sheets were not found on the configuration but it does not mean it was not able to load anything. 

This will even be pointless if we completely remove sheets from configuration, but then we will need to define what sheets to load.

Reply:
ok, let's then just remove MsgBox for now. In principle it's a warning which should be logged somehow; for now we can just ignore it

Reply:
There is a logger implementation in R. This should be logged
See all other services in R where a logger is being used. You can then log a warning or an error